package.json

```diff
@@ -36,8 +36,12 @@
         "rollup"
       ],
       "allowedVersions": {
-        "@solana/web3.js": "1.52.0"
+        "@solana/web3.js": "1.52.0",
```

Comment:
any reason changing `allowedVersions` and `overrides`? Any bugs that we can track to revert that back at some point?

Reply:
this is to silence the pnpm yellow warning you get when it has to make a decision on a version it's not fully happy with. Unfortunately it only works on top level `package.json` even though it's caused by one of the subpackage.

I think the `@solana/web3.js` was caused by a dependency in one of the example, not 100% sure might be `wagmi`.